src/components/PhonicSelector/PhonicSelector.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect, useRef } from "react";
+import phonics from "utils/phonics.json";
+import PhonicIcon from "../PhonicIcon/PhonicIcon";
+import PhonicWordSelector from "../PhonicWordSelector/PhonicWordSelector";
+import ReactCardFlip from "react-card-flip";
+import "../WordGroupSelector/WordGroupSelector.scss";
+import "./PhonicSelector.scss";
+function PhonicSelector(props) {
+  const phonicKeys = useRef(null);
+  const [phonicGroups, setPhonicGroups] = useState({});
+  useEffect(() => {
+    phonicKeys.current = Object.keys(phonics);
+    setPhonicGroups({
+      buh: [phonics[phonicKeys.current[0]]],
+      duh: [phonics[phonicKeys.current[1]]],
+      kuh: [phonics[phonicKeys.current[2]]],
+      fuh: [phonics[phonicKeys.current[3]]]
```

Comment:
i think for this you can make an object by iterating through the phonicKeys, using the phonicKeys as the keys and setting `[phonics[phonicsKey]]` as the value- i'm not too big a fan of doing this sort of manually